packages/deployer/internal/ast/router-ast-validator.js

```diff
@@ -31,12 +31,14 @@ async function findMissingSelectorsInAST(contracts) {
 }
 
 async function findUnreachableSelectorsInAST() {
-  logger.info('  Unreachable selectors not checked yet in compiled router');
```

Comment:
Please, revert those changes that are fixed on my PR so we don't have a collision here.